### ISSUE TITLE
EnableDeveloperMode - broken cproj file

### ISSUE THREAD

**Issue by KSemenenko** [unoplatform/uno#17949]
### Current behavior

I selected developer mode
<img width="382" alt="Screenshot 2024-08-15 at 17 36 14" src="https://github.com/user-attachments/assets/844e9489-8749-4d4d-8f32-1ea416321584">

and my **cproj** file is broken:

```
<ItemGroup>
    $$EnableDeveloperMode_CPM_PackageVersion$$
</ItemGroup>
```

### Works on UWP/WinUI

None

### Environment

Other

### Affected platforms

WebAssembly

### IDE

Rider macOS

### IDE version

JetBrains Rider 2024.2

**Comment by agneszitte** [comment]
Hi @KSemenenko for **Rider 2024.2** you will need to use the [**preview** version](https://plugins.jetbrains.com/plugin/24653-uno-platform/versions/preview/586937) of our plugin at the moment as it is mentioned in our current documentation.
- Setup documentation for Rider: https://aka.platform.uno/rider-getstarted
- Create an Uno Platform app with Rider: https://aka.platform.uno/rider-create-an-app

Make sure to follow the instructions for **Rider 2024.2** for the entire documentation.

We will soon release a stable version of the plugin compatible with 2024.2 now that Rider just released the Rider 2024.2 Stable version. But until then the [**preview** version](https://plugins.jetbrains.com/plugin/24653-uno-platform/versions/preview/586937) of the Uno Platform plugin will need to be used for 2024.2.

**Comment by agneszitte** [comment]
@KSemenenko I will close the issue. But let us know if you have any additional issues with what I shared. 
Thanks in advance for the feedback!
